LayoutTests/fast/loader/navigate-on-pageswap-crash.html

```
@@ -0,0 +1,21 @@
+<!-- webkit-test-runner [ dumpJSConsoleLogInStdErr=true ] -->
+<script>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)